packages/gatsby/src/internal-plugins/query-runner/file-parser.js

```diff
@@ -15,14 +14,6 @@ import type { DocumentNode, DefinitionNode } from "graphql"
 
 const readFileAsync = Bluebird.promisify(fs.readFile)
 
-function getAssignedIdenifier(path) {
```

Comment:
well if no code is calling it whatevers. We can recover it later if it's needed for something.